Maryam-Marjoori/homework_3/3-dice-lab/approach-2/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
+  <head>
+    <title>Dice Roller</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="Your description goes here">
+    <meta name="keywords" content="one, two, three">
+    <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
+    <!-- external CSS link -->
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width">
+  </head>
+
+<!-- ************************************* MODIFIED HTML ************************************* -->
+
+ <body>
+    <div id="container">
+      <header>
+        <h1>Dice Roller</h1>
+      </header>
+      <section id="content">
+        <div> <img id="first-die" src="img/1.png"></div>
+         
+        <div><img  id="second-die" src="img/3.png"></div>
+           
+        <input id="roll-dice" type="button" onclick="roleTheDice()" value="Roll the Dice">
```

Comment:
- not here in HTML, to enforce separation of concerns. Your first attempt was 100% on point. 😸